### PR TITLE
doc: remove the Monitoring Stack documentation from the core ScyllaDB docs

### PR DESCRIPTION
**Pull request by annastuchlik**
I created this branch to remove the external docs (Manager, Monitoring, Operator) from the core ScyllaDB documentation.
However, to make reviewing easier, this PR only covers removing the docs for ScyllaDB Monitoring Stack. I'm going to send other PRs to cover Manager and Operator.

In this PR, I have:
- removed the docs for ScyllaDB Monitoring Stack (including the sources for old versions).
- added redirects to https://monitoring.docs.scylladb.com/.
- replaced the internal links with external links to https://monitoring.docs.scylladb.com/.

### CODE REVIEW

**Comment by tzach** [comment]
I'm getting a few warnings
```
operating-scylla/manager/2.1/add-a-cluster.rst:161: WARNING: unknown document: /operating-scylla/monitoring/index
operating-scylla/manager/2.1/configuration-file.rst:36: WARNING: unknown document: /operating scylla/monitoring/2.2/monitoring-stack
operating-scylla/manager/2.1/install.rst:17: WARNING: unknown document: /operating-scylla/monitoring/2.2/monitoring-stack
operating-scylla/manager/2.1/install.rst:17: WARNING: unknown document: /operating-scylla/monitoring/2.2/monitoring-stack
operating-scylla/manager/2.1/monitoring-manager-integration.rst:7: WARNING: unknown document: /operating scylla/monitoring/index
```

**Comment by annastuchlik** [comment]
Add the link to the Monitoring Stack to the toctree.

**Comment by annastuchlik** [comment]
@tzach I've added the link to ScyllaDB Monitoring Stack to the menu. You can re-review.

File: docs/operating-scylla/index.rst [hunk 1]
```diff
@@ -10,6 +10,7 @@ Scylla for Administrators
    security/index
    admin-tools/index
    manager/index
+   ScyllaDB Monitoring Stack <https://monitoring.docs.scylladb.com/stable/>
```
Comment (tzach) [inline docs/operating-scylla/index.rst:13]: Its better to use <https://monitoring.docs.scylladb.com> without the "stable"

Comment (annastuchlik) [reply docs/operating-scylla/index.rst]: @tzach Corrected.

**Review by tzach — APPROVED** [review]

**Comment by denesb** [comment]
Conflicts, please rebase.

**Comment by annastuchlik** [comment]
@denesb I will. Just one question: How do I know there are conflicts if there is no message above conflicts shown by the checks?

**Comment by denesb** [comment]
> @denesb I will. Just one question: How do I know there are conflicts if there is no message above conflicts shown by the checks?

Github doesn't show whether there are conflicts or not because direct merges to master have been forbidden and seems like Github doesn't think its necessary to show conflict information anymore.

**Comment by annastuchlik** [comment]
@denesb I see, thanks. And thank you for your patience :) I've resolved the conflicts.

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint Please backport to 5.1

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint Please backport to 5.1

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint This PR must be backported to branch-5.1 to prevent the 5.1 documentation build from throwing errors and warnings.

**Comment by denesb** [comment]
Doesn't apply cleanly.

**Pull request closed without merging**